engine/api/workflowtemplate/aggregate_test.go

```diff
@@ -0,0 +1,152 @@
+package workflowtemplate
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/ovh/cds/engine/api/test"
+	"github.com/ovh/cds/sdk"
+)
+
+func TestAggregateAuditsOnWorkflowTemplate(t *testing.T) {
+	db := &test.SqlExecutorMock{}
```

Comment:
so we dont test with the database ? so we don't test the sql queries ?

Reply:
The purpose of this unit test is to check that the AggregateAuditsOnWorkflowTemplate func works so yes there is a mock for the database. I will change the test code for dao functions (Get, GetAll) to do integration tests.